### DISCUSSION TITLE
Prioritising Get-Functions by mapping

### DISCUSSION THREAD

**Discussion by romanschander** [LuckyPennySoftware/AutoMapper#3747] [NA]
Hi there. If i run this test, it fails because they call GetTest-Funktion from Source-Class. How can i prevent it? Regardless of the fact that this is bad programming.

`
        public class Source
        {
            public int Test { get; set; }

            public int GetTest()
            {
                throw new Exception();
            }
        }

        public class Destination
        {
            public int Test { get; set; }
        }

        [Fact]
        public void Test()
        {
            var source = new Source { Test = 1 };

            var config = new MapperConfiguration(cfg =>
            {
                cfg.CreateMap<Source, Destination>();
                
            });
            var mapper = config.CreateMapper();
            var destination = mapper.Map<Source, Destination>(source);
        }`

**Comment by lbargaoanu** [comment]
Try the [MyGet](https://docs.automapper.org/en/latest/The-MyGet-build.html) build.
